Makefile

```diff
@@ -0,0 +1,48 @@
+
+# Common implied folder locations for this build
+outdir=build
+gp211=libs-sdks/jc304_kit/lib/api_classic.jar:libs/globalplatform-2_1_1/gp211.jar
+jars=libs-sdks/jc304_kit/lib/api_classic.jar:$(outdir)/org.idpass.tools.jar 
+
+all: outdir $(outdir)/org/idpass/tools/javacard/tools.cap $(outdir)/org/idpass/auth/javacard/auth.cap $(outdir)/org/idpass/sam/javacard/sam.cap
+	@echo
+	@echo "************************************"
+	@echo "*** LISTING GENERATED CAP FILES ***"
+	@find build/ -type f -name '*.cap' 
+	@echo
+
+outdir:
+	@mkdir -p build/
+
+circleci_env_check:
+	@echo "*** CIRCLECI ENVIRONMENT ***"
+	pwd
+	git remote -v
+	cat .gitmodules
+	ls
+	find . -type f -name '*.java'
+
+$(outdir)/org.idpass.tools.jar: $(outdir)/org/idpass/tools/*.class
+	jar cvf $(outdir)/org.idpass.tools.jar -C $(outdir)/ .
+
+$(outdir)/org/idpass/tools/*.class: src/main/java/org/idpass/tools/*.java
+	javac -source 1.2 -target 1.2 -d $(outdir) -cp $(gp211) src/main/java/org/idpass/tools/*.java
```

Comment:
Would it work if we used 1.8?

Reply:
Starting from 1.8 and going backwards, I find the 1.6 works so I am setting to 1.6. Below is the error when using higher than 1.6:

`parsing /home/circleci/src/card-applets/build/org/idpass/tools/IdpassApplet.class
parsing /home/circleci/src/card-applets/build/org/idpass/tools/SIOAuthListener.class
parsing /home/circleci/src/card-applets/build/org/idpass/tools/Utils.class
error: org.idpass.tools.IdpassApplet: unsupported class file format of version 52.0.
error: org.idpass.tools.SIOAuthListener: unsupported class file format of version 52.0.
error: org.idpass.tools.Utils: unsupported class file format of version 52.0.

[ INFO: ] conversion completed with 3 errors and 0 warnings.
Makefile:47: recipe for target 'build/org/idpass/tools/javacard/tools.cap' failed
`
I took the linux generated `*.cap` files and run `1_SUCCESS_TEST` as an additional verification also.